core.adoc

```diff
@@ -56,7 +56,7 @@ endif::ESSL[]
 :numbered:
 :imagewidth: 800
 :fullimagewidth: width="800"
-:source-highlighter: coderay
+:source-highlighter: rouge
```

Comment:
Looks like this is the missing piece that is stopping it matching Vulkan:
```suggestion
:source-highlighter: rouge
:rouge-style: github
```
If this doesn't break the HTML rendering for you then I think we can make this update and merge the PR.

Reply:
Done, and thank you. Apparently it's the most obvious things that I can't see, heh.